cmd/sync/sync.go

```diff
@@ -109,13 +113,14 @@ func (app *syncApp) start(cmd *cobra.Command, args []string) {
 		Hdist:           app.Config.Hdist,
 		ValidationDelta: 30 * time.Second,
 		LayersPerEpoch:  uint16(app.Config.LayersPerEpoch),
+		GoldenATXID:     goldenATXID,
 	}
 	types.SetLayersPerEpoch(int32(app.Config.LayersPerEpoch))
 	lg.Info("local db path: %v layers per epoch: %v", path, app.Config.LayersPerEpoch)
 
 	if remote {
 		if err := getData(app.Config.DataDir(), version, lg); err != nil {
-			lg.Error("could not download data for test", err)
+			lg.Error("could not download data for test: %v", err)
```

Comment:
As long as we're touching it, we might as well do it like it should be:
```suggestion
			lg.With().Error("could not download data for test", log.Err(err))
```